.env.template

```diff
@@ -1,4 +1,3 @@
 NEXT_PUBLIC_SANITY_PROJECT_ID=
-SANITY_API_TOKEN=
-SANITY_PREVIEW_SECRET=
-TWITTER_API_TOKEN=
+NEXT_PUBLIC_SANITY_TOKEN=
```

Comment:
What I'm proposing regarding having the Sanity + Discord logic to work on a /pages/api function would allow us to just use `SANITY_TOKEN` as the env variable and not have it exposed to the browser.